### ISSUE TITLE
Waybar crashes when swicthing to tty with X session

### ISSUE THREAD

**Issue by roel0** [Alexays/Waybar#346]
Waybar crashes when I move to a tty with a X session running (and back to sway)

Error message:
```
(waybar:4350): GLib-GIO-WARNING **: 19:49:58.479: Received property ToolTip with type s does not match expected type (sa(iiay)ss) in the expected interface

** (waybar:4350): ERROR **: 19:49:58.479: Failed to create DBus Proxy for org.kde.StatusNotifierItem-4065-1 /StatusNotifierItem: std::bad_cast
```

Waybar v0.6.3

**Comment by Alexays** [comment]
Tray issue, @alebastr any idea?

**Comment by alebastr** [comment]
Already fixed in #339. Apparently g_error is a wrong way to log recoverable errors, as it calls abort() internally.

**Comment by Alexays** [comment]
Oh ok got it, thanks!
